src/item.cpp

```diff
@@ -2484,7 +2484,8 @@ void item::food_info( const item *food_item, std::vector<iteminfo> &info,
         parts->test( iteminfo_parts::FOOD_CANNIBALISM ) ) {
         if( !player_character.has_flag( json_flag_CANNIBAL ) &&
             !player_character.has_flag( json_flag_PSYCHOPATH ) &&
-            !player_character.has_flag( json_flag_SAPIOVORE ) ) {
+            !player_character.has_flag( json_flag_SAPIOVORE ) ) && 
+            !player_character.has_flag( json_flag_NUMB ) ) && {
```

Comment:
What I meant is that `NUMB` should not be here, because it still has morale penalty for cannibalism according to the code here:

https://github.com/CleverRaven/Cataclysm-DDA/pull/63745/files#diff-496c4b00def34afbd8adc4d980ed49ce1f05301953d67a436e5813c545b486b1R1266-R1269

Reply:
So I can just delete that part and it's all good?